db.go

```diff
@@ -22,16 +22,16 @@ type DB interface {
 	// Conn only available for the primary db or the first primary db (if using multi-primary)
 	Conn(ctx context.Context) (*sql.Conn, error)
 	Driver() driver.Driver
-	Exec(query string, args ...interface{}) (sql.Result, error)
-	ExecContext(ctx context.Context, query string, args ...interface{}) (sql.Result, error)
+	Exec(query string, args ...any) (sql.Result, error)
```

Comment:
Well I think this is breaking changes, not sure it's a wise decision

Reply:
Minimum go version - 1.18

`any` just a type alias. It should work just fine. 

Argument for that - https://go.dev/play/p/a9dpmUawnsV

Reply:
any is a new [predeclared identifier](https://go.dev/ref/spec#Predeclared_identifiers) and a type alias of interface{}.

It comes from [issue 49884](https://github.com/golang/go/issues/49884), [CL 368254](https://go-review.googlesource.com/c/go/+/368254) and [commit 2580d0e](https://github.com/golang/go/commit/2580d0e08d5e9f979b943758d3c49877fb2324cb).

Reply:
Argument for that - https://github.com/golang/go/issues/49884#issuecomment-987746722

Reply:
Plus even the database/sql is updated to any